packages/cli/src/mud.ts

```diff
@@ -1,8 +1,13 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-esm
```

Comment:
just for context, what's the reason we have to switch to `ts-node` here? Does this mean consumers of the cli need to have ts-node installed?

Reply:
https://github.com/latticexyz/mud/pull/402#discussion_r1117404537 yep. `mud` cli command could be split into 2, so things that don't need config wouldn't need ts-node either. But most things will eventually need config, and 2 cmds sounds bad

Reply:
do you know how much ts node impacts the bundle size? (Old bundle size is 978.82 KB for reference)

Reply:
![Screenshot 2023-02-24 210712](https://user-images.githubusercontent.com/62725447/221256317-a7597474-69fe-4b1b-99b2-7bc1d1e1dd15.png)

Reply:
looks like surprisingly little